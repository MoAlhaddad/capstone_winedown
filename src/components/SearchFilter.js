import React, { useState, useEffect } from "react";
import WineApi from "../api/wineapi";

const filterItems = [
  { key: 1, label: "vintage" },
  { key: 2, label: "gws" },
  {
    key: 3,
    label: "ci",
  },
  {
    key: 4,
    label: "nbj",
  },
  {
    key: 5,
    label: "country",
  },
];

export default function SearchFilter({ wineList, setWinelist }) {
  const [filter, setFilterForm] = useState({
    propertyToFilter: filterItems[0].label,
    filterData: "",
  });
  const handleChange = (evt) => {
    const target = evt.target;
    const value = target.type == "checkbox" ? target.checked : target.value;
    const copyOfFilter = Object.assign({}, filter);
    copyOfFilter.filterData = value;
    setFilterForm(copyOfFilter);
  };

  const handleSelectChange = (evt) => {
    const target = evt.target;
    const value = target.value;
    const copyOfFilter = Object.assign({}, filter);
    copyOfFilter.propertyToFilter = value;
    setFilterForm(copyOfFilter);
  };

  const onFilter = async (evt) => {
    evt.preventDefault();
    const wineResponse = await WineApi.filterWines(filter);
    setWinelist(wineResponse.filteredWines);
  };

  const resetToOriginalWineList = async (evt) => {
    evt.preventDefault();
    const wineResponse = await WineApi.getWines();
    setFilterForm({
      propertyToFilter: filterItems[0].label,
      filterData: "",
    });
    setWinelist(wineResponse.wines);
  };

  return (
    <form style={{ display: "flex" }} onSubmit={onFilter}>
      <div id={"search-filter"} style={{ display: "flex" }}>
        <input
          id="searchValue"
          name="searchValue"
          onChange={handleChange}
          placeholder="Filter winedown list"
          value={filter.filterData}
          style={{ width: "70%" }}
        />
        <select
          id={"propertyToSearchFor"}
          style={{ width: "30%" }}
          onChange={handleSelectChange}
          defaultValue={filter.propertyToFilter}
          value={filter.propertyToFilter}
        >
          {filterItems.map((filterItem) => {
            return (
              <option key={filterItem.key} value={filterItem.label}>
                {filterItem.label}
              </option>
            );
          })}
        </select>
      </div>
      <button
        type="submit"
        className="btn btn-primary"
        style={{ width: "120px" }}
      >
        Filter
      </button>
      <button
        type="button"
        className="btn btn-secondary"
        style={{ width: "120px" }}
        onClick={resetToOriginalWineList}
      >
        Reset
      </button>
    </form>
  );
}
