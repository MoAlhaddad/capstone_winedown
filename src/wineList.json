{
    "data": [
      {
        "id": 1,
        "wine": "Domaine De La Romanee Conti,Romanee Conti Grand Cru",
        "vintage": 2005,
        "gws": "100.00",
        "ci": "A+",
        "nbj": 4,
        "country_id": 1,
        "date_created": "2021-10-04T04:18:47.445Z",
        "date_updated": null,
        "deleted_ind": null,
        "is_favorite_wine": null,
        "country": "France"
      },
      {
        "id": 3,
        "wine": "Domaine Leroy, Romanee Saint Vivant Grand Cru",
        "vintage": 2015,
        "gws": "100.00",
        "ci": "A+",
        "nbj": 6,
        "country_id": 1,
        "date_created": "2021-10-04T04:20:48.336Z",
        "date_updated": null,
        "deleted_ind": null,
        "is_favorite_wine": null,
        "country": "France"
      },
      {
        "id": 4,
        "wine": "Leroy Domaine D'Auvenay, Blanc, Chevalier Montrachet Grand Cru",
        "vintage": 2002,
        "gws": "100.00",
        "ci": "A+",
        "nbj": 3,
        "country_id": 1,
        "date_created": "2021-10-04T04:20:52.439Z",
        "date_updated": null,
        "deleted_ind": null,
        "is_favorite_wine": null,
        "country": "France"
      },
      {
        "id": 5,
        "wine": "Chateau Lafite Rothschild, Pauillac",
        "vintage": 1953,
        "gws": "100.00",
        "ci": "B",
        "nbj": 3,
        "country_id": 1,
        "date_created": "2021-10-04T04:21:03.649Z",
        "date_updated": null,
        "deleted_ind": null,
        "is_favorite_wine": null,
        "country": "France"
      },
      {
        "id": 6,
        "wine": "Domaine Georges & Christophe Roumier, Musigny Grand Cru",
        "vintage": 2005,
        "gws": "100.00",
        "ci": "A",
        "nbj": 3,
        "country_id": 1,
        "date_created": "2021-10-04T04:21:06.920Z",
        "date_updated": null,
        "deleted_ind": null,
        "is_favorite_wine": null,
        "country": "France"
      },
      {
        "id": 7,
        "wine": "Chateau Lafleur, Pomerol",
        "vintage": 1950,
        "gws": "100.00",
        "ci": "A+",
        "nbj": 4,
        "country_id": 1,
        "date_created": "2021-10-04T04:21:11.655Z",
        "date_updated": null,
        "deleted_ind": null,
        "is_favorite_wine": null,
        "country": "France"
      },
      {
        "id": 8,
        "wine": "Leroy Domaine D'Auvenay, Blanc, Chevalier Montrachet Grand Cru",
        "vintage": 2008,
        "gws": "100.00",
        "ci": "A",
        "nbj": 3,
        "country_id": 1,
        "date_created": "2021-10-04T04:21:56.759Z",
        "date_updated": null,
        "deleted_ind": null,
        "is_favorite_wine": null,
        "country": "France"
      },
      {
        "id": 9,
        "wine": "Chateau D'Yquem, Blanc, Sauternes",
        "vintage": 1811,
        "gws": "100.00",
        "ci": "A",
        "nbj": 3,
        "country_id": 1,
        "date_created": "2021-10-04T04:22:07.799Z",
        "date_updated": null,
        "deleted_ind": null,
        "is_favorite_wine": null,
        "country": "France"
      },
      {
        "id": 10,
        "wine": "Domaine De La Romanee Conti, Romanee Conti Grand Cru",
        "vintage": 1999,
        "gws": "100.00",
        "ci": "A+",
        "nbj": 3,
        "country_id": 1,
        "date_created": "2021-10-04T04:22:36.975Z",
        "date_updated": null,
        "deleted_ind": null,
        "is_favorite_wine": null,
        "country": "France"
      },
      {
        "id": 11,
        "wine": "Domaine De La Romanee Conti, Romanee Conti Grand Cru",
        "vintage": 2015,
        "gws": "100.00",
        "ci": "A+",
        "nbj": 6,
        "country_id": 1,
        "date_created": "2021-10-04T04:22:42.517Z",
        "date_updated": null,
        "deleted_ind": null,
        "is_favorite_wine": null,
        "country": "France"
      },
      {
        "id": 12,
        "wine": "Chateau D'Yquem, Blanc, Sauternes",
        "vintage": 1847,
        "gws": "100.00",
        "ci": "A+",
        "nbj": 3,
        "country_id": 1,
        "date_created": "2021-10-04T04:22:47.899Z",
        "date_updated": null,
        "deleted_ind": null,
        "is_favorite_wine": null,
        "country": "France"
      },
      {
        "id": 13,
        "wine": "Domaine De La Romanee Conti, Romanee Conti Grand Cru",
        "vintage": 2012,
        "gws": "100.00",
        "ci": "A",
        "nbj": 4,
        "country_id": 1,
        "date_created": "2021-10-04T04:23:26.823Z",
        "date_updated": null,
        "deleted_ind": null,
        "is_favorite_wine": null,
        "country": "France"
      },
      {
        "id": 14,
        "wine": "Domaine De La Romanee Conti, La Tache Grand Cru",
        "vintage": 1999,
        "gws": "100.00",
        "ci": "B",
        "nbj": 3,
        "country_id": 1,
        "date_created": "2021-10-04T04:23:26.823Z",
        "date_updated": null,
        "deleted_ind": null,
        "is_favorite_wine": null,
        "country": "France"
      },
      {
        "id": 15,
        "wine": "Domaine Leroy, Musigny Grand Cru",
        "vintage": 2010,
        "gws": "100.00",
        "ci": "A",
        "nbj": 3,
        "country_id": 1,
        "date_created": "2021-10-04T04:23:26.823Z",
        "date_updated": null,
        "deleted_ind": null,
        "is_favorite_wine": null,
        "country": "France"
      },
      {
        "id": 16,
        "wine": "Coche Dury, Blanc, Corton Charlemagne Grand Cru",
        "vintage": 2014,
        "gws": "99.97",
        "ci": "A+",
        "nbj": 3,
        "country_id": 1,
        "date_created": "2021-10-04T04:23:26.823Z",
        "date_updated": null,
        "deleted_ind": null,
        "is_favorite_wine": null,
        "country": "France"
      },
      {
        "id": 17,
        "wine": "Domaine De La Romanee Conti, Romanee Conti Grand Cru",
        "vintage": 2010,
        "gws": "99.85",
        "ci": "B+",
        "nbj": 5,
        "country_id": 1,
        "date_created": "2021-10-04T04:23:26.823Z",
        "date_updated": null,
        "deleted_ind": null,
        "is_favorite_wine": null,
        "country": "France"
      },
      {
        "id": 18,
        "wine": "Leroy Domaine D'Auvenay, Bonnes Mares Grand Cru",
        "vintage": 2012,
        "gws": "99.82",
        "ci": "A+",
        "nbj": 3,
        "country_id": 1,
        "date_created": "2021-10-04T04:23:26.823Z",
        "date_updated": null,
        "deleted_ind": null,
        "is_favorite_wine": null,
        "country": "France"
      },
      {
        "id": 19,
        "wine": "Domaine Leroy, Chambertin Grand Cru",
        "vintage": 2010,
        "gws": "99.82",
        "ci": "A+",
        "nbj": 4,
        "country_id": 1,
        "date_created": "2021-10-04T04:23:43.730Z",
        "date_updated": null,
        "deleted_ind": null,
        "is_favorite_wine": null,
        "country": "France"
      },
      {
        "id": 20,
        "wine": "Masseto, Toscana",
        "vintage": 2015,
        "gws": "100.00",
        "ci": "A+",
        "nbj": 3,
        "country_id": 2,
        "date_created": "2021-10-04T04:26:48.448Z",
        "date_updated": null,
        "deleted_ind": null,
        "is_favorite_wine": null,
        "country": "Italy"
      },
      {
        "id": 21,
        "wine": "Masseto, Toscana",
        "vintage": 2016,
        "gws": "99.98",
        "ci": "A+",
        "nbj": 3,
        "country_id": 2,
        "date_created": "2021-10-04T04:26:48.448Z",
        "date_updated": null,
        "deleted_ind": null,
        "is_favorite_wine": null,
        "country": "Italy"
      },
      {
        "id": 22,
        "wine": "Marchesi Antinori, Tenuta Tignanello, Solaia, Toscana",
        "vintage": 2015,
        "gws": "99.82",
        "ci": "A+",
        "nbj": 4,
        "country_id": 2,
        "date_created": "2021-10-04T04:26:48.448Z",
        "date_updated": null,
        "deleted_ind": null,
        "is_favorite_wine": null,
        "country": "Italy"
      },
      {
        "id": 23,
        "wine": "Masseto, Toscana",
        "vintage": 2015,
        "gws": "100.00",
        "ci": "A+",
        "nbj": 3,
        "country_id": 2,
        "date_created": "2021-10-04T04:26:48.719Z",
        "date_updated": null,
        "deleted_ind": null,
        "is_favorite_wine": null,
        "country": "Italy"
      },
      {
        "id": 24,
        "wine": "Masseto, Toscana",
        "vintage": 2016,
        "gws": "99.98",
        "ci": "A+",
        "nbj": 3,
        "country_id": 2,
        "date_created": "2021-10-04T04:26:48.719Z",
        "date_updated": null,
        "deleted_ind": null,
        "is_favorite_wine": null,
        "country": "Italy"
      },
      {
        "id": 25,
        "wine": "Marchesi Antinori, Tenuta Tignanello, Solaia, Toscana",
        "vintage": 2015,
        "gws": "99.82",
        "ci": "A+",
        "nbj": 4,
        "country_id": 2,
        "date_created": "2021-10-04T04:26:48.719Z",
        "date_updated": null,
        "deleted_ind": null,
        "is_favorite_wine": null,
        "country": "Italy"
      },
      {
        "id": 26,
        "wine": "Dominio De Pingus, Ribera Del Duero",
        "vintage": 2014,
        "gws": "99.24",
        "ci": "B+",
        "nbj": 3,
        "country_id": 7,
        "date_created": "2021-10-04T04:27:09.474Z",
        "date_updated": null,
        "deleted_ind": null,
        "is_favorite_wine": null,
        "country": "Spain"
      },
      {
        "id": 27,
        "wine": "Dominio De Pingus, Ribera Del Duero",
        "vintage": 2012,
        "gws": "98.15",
        "ci": "B",
        "nbj": 4,
        "country_id": 7,
        "date_created": "2021-10-04T04:27:09.474Z",
        "date_updated": null,
        "deleted_ind": null,
        "is_favorite_wine": null,
        "country": "Spain"
      },
      {
        "id": 28,
        "wine": "Dominio De Pingus, Ribera Del Duero",
        "vintage": 2004,
        "gws": "98.00",
        "ci": "B",
        "nbj": 3,
        "country_id": 7,
        "date_created": "2021-10-04T04:27:09.474Z",
        "date_updated": null,
        "deleted_ind": null,
        "is_favorite_wine": null,
        "country": "Spain"
      },
      {
        "id": 29,
        "wine": "Dominio De Pingus, Ribera Del Duero",
        "vintage": 2014,
        "gws": "99.24",
        "ci": "B+",
        "nbj": 3,
        "country_id": 7,
        "date_created": "2021-10-04T04:27:10.312Z",
        "date_updated": null,
        "deleted_ind": null,
        "is_favorite_wine": null,
        "country": "Spain"
      },
      {
        "id": 30,
        "wine": "Dominio De Pingus, Ribera Del Duero",
        "vintage": 2012,
        "gws": "98.15",
        "ci": "B",
        "nbj": 4,
        "country_id": 7,
        "date_created": "2021-10-04T04:27:10.312Z",
        "date_updated": null,
        "deleted_ind": null,
        "is_favorite_wine": null,
        "country": "Spain"
      },
      {
        "id": 31,
        "wine": "Dominio De Pingus, Ribera Del Duero",
        "vintage": 2004,
        "gws": "98.00",
        "ci": "B",
        "nbj": 3,
        "country_id": 7,
        "date_created": "2021-10-04T04:27:10.312Z",
        "date_updated": null,
        "deleted_ind": null,
        "is_favorite_wine": null,
        "country": "Spain"
      },
      {
        "id": 32,
        "wine": "Dominio De Pingus, Ribera Del Duero",
        "vintage": 2014,
        "gws": "99.24",
        "ci": "B+",
        "nbj": 3,
        "country_id": 7,
        "date_created": "2021-10-04T04:27:10.518Z",
        "date_updated": null,
        "deleted_ind": null,
        "is_favorite_wine": null,
        "country": "Spain"
      },
      {
        "id": 33,
        "wine": "Dominio De Pingus, Ribera Del Duero",
        "vintage": 2012,
        "gws": "98.15",
        "ci": "B",
        "nbj": 4,
        "country_id": 7,
        "date_created": "2021-10-04T04:27:10.518Z",
        "date_updated": null,
        "deleted_ind": null,
        "is_favorite_wine": null,
        "country": "Spain"
      },
      {
        "id": 34,
        "wine": "Dominio De Pingus, Ribera Del Duero",
        "vintage": 2004,
        "gws": "98.00",
        "ci": "B",
        "nbj": 3,
        "country_id": 7,
        "date_created": "2021-10-04T04:27:10.518Z",
        "date_updated": null,
        "deleted_ind": null,
        "is_favorite_wine": null,
        "country": "Spain"
      },
      {
        "id": 35,
        "wine": "Test Created Wine",
        "vintage": 2021,
        "gws": "100.00",
        "ci": "A+",
        "nbj": 3,
        "country_id": 4,
        "date_created": "2021-11-26T11:05:06.414Z",
        "date_updated": null,
        "deleted_ind": null,
        "is_favorite_wine": null,
        "country": "Argentina"
      },
      {
        "id": 36,
        "wine": "Test Created Wine 2",
        "vintage": 2021,
        "gws": "90.00",
        "ci": "A",
        "nbj": 4,
        "country_id": 10,
        "date_created": "2021-11-26T11:05:57.640Z",
        "date_updated": null,
        "deleted_ind": null,
        "is_favorite_wine": null,
        "country": "USA"
      },
      {
        "id": 37,
        "wine": "Test Created Wine 3",
        "vintage": 2021,
        "gws": "90.00",
        "ci": "A",
        "nbj": 4,
        "country_id": 3,
        "date_created": "2021-11-26T11:08:23.487Z",
        "date_updated": null,
        "deleted_ind": null,
        "is_favorite_wine": null,
        "country": "Sicily"
      },
      {
        "id": 38,
        "wine": "Bill Gates ",
        "vintage": 4,
        "gws": "90.00",
        "ci": "100",
        "nbj": 2005,
        "country_id": 2,
        "date_created": "2021-12-06T10:08:29.134Z",
        "date_updated": null,
        "deleted_ind": null,
        "is_favorite_wine": null,
        "country": "Italy"
      },
      {
        "id": 39,
        "wine": "Domaine Leroy",
        "vintage": 4,
        "gws": "90.00",
        "ci": "100",
        "nbj": 6,
        "country_id": 7,
        "date_created": "2021-12-10T10:05:03.317Z",
        "date_updated": null,
        "deleted_ind": null,
        "is_favorite_wine": null,
        "country": "Spain"
      },
      {
        "id": 40,
        "wine": "Lebrons blood",
        "vintage": 4,
        "gws": "90.00",
        "ci": "",
        "nbj": 6,
        "country_id": 7,
        "date_created": "2021-12-10T10:06:04.362Z",
        "date_updated": null,
        "deleted_ind": null,
        "is_favorite_wine": null,
        "country": "Spain"
      },
      {
        "id": 41,
        "wine": "Lebrons blood",
        "vintage": 4,
        "gws": "90.00",
        "ci": "",
        "nbj": 6,
        "country_id": 7,
        "date_created": "2021-12-10T10:06:04.678Z",
        "date_updated": null,
        "deleted_ind": null,
        "is_favorite_wine": null,
        "country": "Spain"
      },
      {
        "id": 42,
        "wine": "Mos juice",
        "vintage": 2004,
        "gws": "90.00",
        "ci": "A+",
        "nbj": 4,
        "country_id": 7,
        "date_created": "2021-12-10T10:20:40.023Z",
        "date_updated": null,
        "deleted_ind": null,
        "is_favorite_wine": true,
        "country": "Spain"
      },
      {
        "id": 43,
        "wine": "Kobes Blood",
        "vintage": 2004,
        "gws": "100.00",
        "ci": "A+",
        "nbj": 4,
        "country_id": 2,
        "date_created": "2021-12-10T10:34:11.742Z",
        "date_updated": null,
        "deleted_ind": null,
        "is_favorite_wine": true,
        "country": "Italy"
      },
      {
        "id": 44,
        "wine": "Ronaldos Blood",
        "vintage": 2004,
        "gws": "100.01",
        "ci": "A+",
        "nbj": 4,
        "country_id": 2,
        "date_created": "2021-12-10T10:38:40.933Z",
        "date_updated": null,
        "deleted_ind": null,
        "is_favorite_wine": true,
        "country": "Italy"
      },
      {
        "id": 45,
        "wine": "Chateau Margaux",
        "vintage": 2003,
        "gws": "94.80",
        "ci": "A",
        "nbj": 5,
        "country_id": 1,
        "date_created": "2021-12-19T07:32:08.291Z",
        "date_updated": null,
        "deleted_ind": null,
        "is_favorite_wine": null,
        "country": "France"
      },
      {
        "id": 46,
        "wine": "Ballin juice",
        "vintage": 2003,
        "gws": "94.80",
        "ci": "A",
        "nbj": 5,
        "country_id": 12,
        "date_created": "2021-12-19T07:33:55.439Z",
        "date_updated": null,
        "deleted_ind": null,
        "is_favorite_wine": null,
        "country": "China"
      },
      {
        "id": 47,
        "wine": "Sayain blood",
        "vintage": 2001,
        "gws": "94.80",
        "ci": "A",
        "nbj": 4,
        "country_id": 2,
        "date_created": "2021-12-19T07:38:02.487Z",
        "date_updated": null,
        "deleted_ind": null,
        "is_favorite_wine": null,
        "country": "Italy"
      },
      {
        "id": 48,
        "wine": "Test Created wine 3",
        "vintage": 2021,
        "gws": "10.00",
        "ci": "A+",
        "nbj": 3,
        "country_id": 3,
        "date_created": "2022-01-02T04:25:30.359Z",
        "date_updated": null,
        "deleted_ind": null,
        "is_favorite_wine": false,
        "country": "Sicily"
      },
      {
        "id": 49,
        "wine": "Red Wine",
        "vintage": 2000,
        "gws": "90.00",
        "ci": "A",
        "nbj": 3,
        "country_id": 5,
        "date_created": "2022-01-02T07:18:00.749Z",
        "date_updated": null,
        "deleted_ind": null,
        "is_favorite_wine": true,
        "country": "Portugal"
      },
      {
        "id": 50,
        "wine": "Test Wine 2022",
        "vintage": 2019,
        "gws": "98.00",
        "ci": "A",
        "nbj": 3,
        "country_id": 7,
        "date_created": "2022-01-02T07:21:03.004Z",
        "date_updated": null,
        "deleted_ind": null,
        "is_favorite_wine": true,
        "country": "Spain"
      },
      {
        "id": 51,
        "wine": "Lebron Wine",
        "vintage": 2000,
        "gws": "90.00",
        "ci": "A",
        "nbj": 3,
        "country_id": 1,
        "date_created": "2022-01-02T07:29:03.097Z",
        "date_updated": null,
        "deleted_ind": null,
        "is_favorite_wine": true,
        "country": "France"
      },
      {
        "id": 52,
        "wine": "Domaine Leroy, Romanee Saint Vivant Grand Cru",
        "vintage": 2001,
        "gws": "100.00",
        "ci": "A",
        "nbj": 4,
        "country_id": 1,
        "date_created": "2022-01-03T05:29:31.076Z",
        "date_updated": null,
        "deleted_ind": null,
        "is_favorite_wine": true,
        "country": "France"
      }
    ]
  }
  