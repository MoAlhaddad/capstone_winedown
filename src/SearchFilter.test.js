import * as React from 'react';
//import testing utilities
import SearchFilter from "./components/SearchFilter";
import wineList from "./wineList.json";
import renderer from 'react-test-renderer';

let testWineList = [];
describe("SearchFilter", () => {
  it("rendered SearchFilter", () => {
    const setWineList = (wines) => { stupidWineList = wines };
    const component = renderer.create(
     <SearchFilter wineList={wineList.data} setWinelist={setWineList} />
    );
    expect(component).toMatchSnapshot();
  });
});

