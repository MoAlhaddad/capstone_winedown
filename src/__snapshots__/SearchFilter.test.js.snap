// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchFilter rendered SearchFilter 1`] = `
<form
  onSubmit={[Function]}
  style={
    Object {
      "display": "flex",
      "justifyContent": "space-between",
    }
  }
>
  <div
    id="search-filter"
    style={
      Object {
        "display": "flex",
      }
    }
  >
    <div
      style={
        Object {
          "display": "flex",
          "marginRight": "10px",
        }
      }
    >
      <button
        className="btn not-active"
        id={1}
        onClick={[Function]}
        type="button"
      >
        vintage
      </button>
    </div>
    <div
      style={
        Object {
          "display": "flex",
          "marginRight": "10px",
        }
      }
    >
      <button
        className="btn not-active"
        id={2}
        onClick={[Function]}
        type="button"
      >
        gws
      </button>
    </div>
    <div
      style={
        Object {
          "display": "flex",
          "marginRight": "10px",
        }
      }
    >
      <button
        className="btn not-active"
        id={3}
        onClick={[Function]}
        type="button"
      >
        ci
      </button>
    </div>
    <div
      style={
        Object {
          "display": "flex",
          "marginRight": "10px",
        }
      }
    >
      <button
        className="btn not-active"
        id={4}
        onClick={[Function]}
        type="button"
      >
        nbj
      </button>
    </div>
    <div
      style={
        Object {
          "display": "flex",
          "marginRight": "10px",
        }
      }
    >
      <button
        className="btn not-active"
        id={5}
        onClick={[Function]}
        type="button"
      >
        country
      </button>
    </div>
    <div>
      And
      <br />
       Comparison
      <input
        checked={false}
        onChange={[Function]}
        type="Checkbox"
      />
    </div>
  </div>
  <div>
    <button
      className="btn btn-primary"
      style={
        Object {
          "width": "120px",
        }
      }
      type="submit"
    >
      Filter
    </button>
    <button
      className="btn btn-secondary"
      onClick={[Function]}
      style={
        Object {
          "width": "120px",
        }
      }
      type="button"
    >
      Reset
    </button>
  </div>
</form>
`;
